From a89111781515d248159b53b517330c7aa2dbaea6 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 22 Nov 2016 07:34:45 -0500
Subject: [PATCH 12/23] always use the WebView sandbox

---
 .../chromium/WebViewChromiumFactoryProvider.java     | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
index ebdc7e0fb43c..3204f3ea1027 100644
--- a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
+++ b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
@@ -228,17 +228,7 @@ public class WebViewChromiumFactoryProvider implements WebViewFactoryProvider {
                 CommandLineUtil.initCommandLine();
             }
 
-            boolean multiProcess = false;
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-                // Ask the system if multiprocess should be enabled on O+.
-                multiProcess = mWebViewDelegate.isMultiProcessEnabled();
-            } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-                // Check the multiprocess developer setting directly on N.
-                multiProcess = Settings.Global.getInt(
-                                       ContextUtils.getApplicationContext().getContentResolver(),
-                                       Settings.Global.WEBVIEW_MULTIPROCESS, 0)
-                        == 1;
-            }
+            boolean multiProcess = true;
             if (multiProcess) {
                 CommandLine cl = CommandLine.getInstance();
                 cl.appendSwitch("webview-sandboxed-renderer");
-- 
2.18.0

